src/algebra/algebra/basic.lean

```diff
@@ -1312,6 +1312,11 @@ variables {R}
 instance int_algebra_subsingleton : subsingleton (algebra ℤ R) :=
 ⟨λ P Q, by { ext, simp, }⟩
 
+
+def int_linear_equiv_of_add_equiv {α β : Type*} [add_comm_group α] [add_comm_group β]
+  (e : α ≃+ β) : α ≃ₗ[ℤ] β :=
+e.to_linear_equiv $ λ c a, by { erw e.to_add_monoid_hom.map_zsmul, refl }
```

Comment:
This belongs in `data/equiv/module` like I said on Zulip ;)

Can you call it `add_equiv.to_int_linear_equiv` instead?

Reply:
Sorry, missed that line on zulip. Will do.

Reply:
How about I’ll move it to `linear_algebra.basic`, next to `add_equiv.to_linear_equiv`?

Reply:
Also not good, there instances for `module ℤ α` are missing.

Reply:
`linear_algebra.basic` imports `algebra.module.basic` so should be fine.